docs/tutorial/force_calibration.rst

```diff
@@ -212,3 +212,102 @@ If we compare the different fits, we can see that the Lorentzian model with diod
+Using active calibration, the accuracy of the calibration can be improved, because active calibration uses fewer assumptions than passive calibration.
+
+When performing passive calibration, we base our calculations on a theoretical drag coefficient.
+This theoretical drag coefficient depends on parameters that are only known with limited precision: the diameter of the bead and the viscosity.
+This viscosity in turn depends strongly on the local temperature around the bead, which is typically poorly known.
+
+During active calibration, the trap or nanostage is oscillated sinusoidally.
+These oscillations result in a driving peak in the force spectrum.
+Using power spectral analysis, the force can then be calibrated without prior knowledge of the drag coefficient.
+
+When the power spectrum is computed from an integer number of oscillations, the driving peak is visible at a single data point at :math:`f_{drive}`.
+
+.. image:: driving_input.png
+
+The physical spectrum is then given by a thermal part (like before) and an active part:
+
+.. math::
+
+    P^{thermal}(f) = \frac{D}{\pi ^ 2 \left(f^2 + f_c ^ 2\right)}
+
+    P^{active}(f) = \frac{A^2}{2\left(1 + \frac{f_c^2}{f_{drive}^2}\right)} \delta(f - f_{drive})
+
+    P^{total}(f) = P^{thermal}(f) + P^{active}(f)
+
+Since we know the driving amplitude, :math:`A`, we know how the bead reacts to the driving motion and we can observe this response in the power spectral density (PSD), we can use this relation to determine the positional calibration.
+
+If we use the basic Lorentzian model, then the theoretical power (integral over the delta spike) corresponding to the driving input is given by :cite:`tolic2006calibration`:
+
+.. math::
+
+    W_{physical} = \frac{A^2}{2\left(1 + \frac{f_c^2}{f_{drive}^2}\right)} \quad \mathrm{[m^2]}
+
+Subtracting the thermal part of the spectrum, we can determine the same quantity experimentally.
+
+.. math::
+
+    W_{measured} = \left(P_{measured}^{total}(f_{drive}) - P_{measured}^{thermal}(f_{drive})\right) \Delta f \quad \mathrm{[V^2]}
+
+:math:`\Delta f` refers to the width of one spectral bin.
+Here the thermal contribution that needs to be subtracted is obtained from fitting the thermal part of the spectrum using the passive calibration procedure from before.
+The desired positional calibration is then:
+
+.. math::
+
+    R_d = \sqrt{\frac{W_{physical}}{W_{measured}}} \quad \mathrm{[m/V]}
+
+Note how this time around, we did not rely on assumptions on the viscosity of the medium or the bead size.
+
+As a side effect of this calibration, we actually obtain an experimental estimate of the drag coefficient:
+
+.. math::
+
+    \gamma_{measured} = \frac{k_B T}{R_d^2 D_{measured}} \quad \mathrm{[kg/s]}
+
+How to do active calibration in Pylake
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Using `Pylake`, the procedure to use active calibration is not very different from passive calibration.
```

Comment:
`pylake` should be lowercase since we're referring to the package (code-formatted)

Reply:
👍